completion/available/npm.completion.bash

```diff
@@ -1,8 +1,9 @@
 #!/usr/bin/env bash
 
 # npm (Node Package Manager) completion
+# https://docs.npmjs.com/cli/completion
 
 if command -v npm &>/dev/null
 then
-  eval "$(npm completion)"
+  source <(npm completion)
```

Comment:
Yes, that's right - I use Bash v3.x. The old command works fine for me, no issues. Thanks for fixing this!